Project.toml

```diff
@@ -12,7 +12,7 @@ MuladdMacro = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
-ERFA = "^0.4"
+ERFA = "^0"
```

Comment:
This is not acceptable, this includes an infinite number of breaking changes, you must list all of them explicitely